apps/web2/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -3,8 +3,8 @@ import { Link, useHistory } from 'react-router-dom';
 
 import { useUserContext } from '../../contexts/UserContext';
 
-import logo from '../../assets/wya-logo.png';
-import image from '../../assets/background9.png';
+// import logo from '../../assets/wya-logo.png';
+import image from '../../assets/background14.png';
```

Comment:
Is there a reason we are naming them `background14`? Do we have multiple backgrounds? Could we just name it background?

Reply:
It's so that I can keep track of which changes I have. I have had to make multiple changes to the background as far as sharpness, idea, etc. I'd rather keep the name to background14 so that I don't lose track of which image we're using.

Reply:
Okay makes sense

Reply:
Also I would prefer to keep the commented code as reference to tailwind. I didn't want to use certain aspects in the Landing Page but feel that it will be useful in other pages. It's good to keep the commented code for the logo and links just so that we all can use it as reference when using tailwind for the other pages

Reply:
Okay seems like a good reason for doing so. So can you please make an annotation above the code you want to keep in via a comment along the lines of `Leaving this here a reference`.

Want to make it clear to others why the code is left in, otherwise commented code should be deleted, because we can always look in git history if we really need to.